internal/infrastructure/kubernetes/ratelimit/resource.go

```diff
@@ -399,3 +461,31 @@ func Validate(ctx context.Context, client client.Client, gateway *egv1a1.EnvoyGa
 
 	return nil
 }
+
+func enableTracing(rl *egv1a1.RateLimit) bool {
+	// Other fields can use the default values,
+	// but we have to make sure the user has the Provider.URL
+	if rl != nil && rl.Telemetry != nil &&
+		rl.Telemetry.Tracing != nil &&
+		rl.Telemetry.Tracing.Provider != nil &&
+		len(rl.Telemetry.Tracing.Provider.URL) != 0 {
+		return true
+	}
+
+	return false
+}
+
+// checkTraceEndpointScheme Check the scheme prefix in the trace url
```

Comment:
is this needed ? afaik, are url w/o schemes not working ?

Reply:
Yes, it cannot take effect without configuring the scheme.